website/docs/getting_started/about.md

```diff
@@ -1,10 +1,10 @@
 ---
-id: overview
+id: about
 title: What is Meshtastic?
-sidebar_label: Overview
-slug: /getting_started
+sidebar_label: About
+slug: /about
```

Comment:
This breaks the navigation:

Header points to `docs/getting_started`
this page isn't nested in the `getting_started` dir when it should be, so header highlighting doesn't work correctly

Reply:
You're correct. This was the beginning or abstracting the `about` page away from the `getting_started` section. This does break the navigation, however, below I've created a new overview file that is nested in the `getting_started` folder, located at `docs/getting_started` On a future PR we will remove this about file completely once the about page has been built out. (Outside of the docs page) See the comment in this file on line 7, indicating further changes needed.

Navigation does work properly, and soon we will remove `about` from the sidebar altogether.
https://meshtastic-jfirwin.vercel.app/docs/about
https://meshtastic-jfirwin.vercel.app/docs/getting_started

Reply:
Perfect, we'll remove once we embed the last of the info from about into their respective pages